nms/app/views/subscriber/SubscriberAddDialog.tsx

```diff
@@ -110,36 +92,29 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const forbiddenNetworkTypes = Object.keys(CoreNetworkTypes).map(
-  key => CoreNetworkTypes[key],
-);
+const forbiddenNetworkTypes = Object.values(CoreNetworkTypes);
 
 type ActionDialogProps = {
-  open: boolean,
-  onClose: () => void,
-  editProps?: EditProps,
+  open: boolean;
+  onClose: () => void;
+  editProps?: EditProps;
   onSave: (
     subscribers: Array<SubscriberInfo>,
     selectedSubscribers?: Array<string>,
-  ) => void,
-  error?: string,
-  subscriberAction: SubscriberActionType,
+  ) => void;
+  error?: string;
+  subscriberAction: SubscriberActionType;
 };
 
 /**
  * Dialog used to Add/Delete/Update subscribers
  */
 export function AddSubscriberDialog(props: ActionDialogProps) {
   const classes = useStyles();
+
   return (
     <>
-      <Dialog
-        data-testid="addSubscriberDialog"
-        open={props.open}
-        onSave={(subscribers, selectedSubscribers) => {
-          props.onSave?.(subscribers || [], selectedSubscribers);
-        }}
```

Comment:
Yes this confused me for a while. The generic Matrial-UI dialog does not have an `onSave` callback and most certainly doesn't know anything about subscribers.
`onSave` is also passed to `SubscriberDetailsDialogContent` where it will actually be used.